Braintree.xcodeproj/xcshareddata/xcschemes/UnitTests.xcscheme

```diff
@@ -173,6 +173,16 @@
                ReferencedContainer = "container:Braintree.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "8046983D2B27C5530090878E"
+               BuildableName = "BraintreeShopperInsightsTests.xctest"
+               BlueprintName = "BraintreeShopperInsightsTests"
+               ReferencedContainer = "container:Braintree.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
       </Testables>
```

Comment:
what does this do, Jax?

Reply:
The `BraintreeShopperInsightsTests` were listed under this section:
![Screenshot 2024-06-06 at 2 25 52 PM](https://github.com/braintree/braintree_ios/assets/20733831/fa9acc57-5907-4776-9354-6e3d6c8f5b77)

This change adds them under this section with the rest of our Unit Tests:
![Screenshot 2024-06-06 at 2 26 51 PM](https://github.com/braintree/braintree_ios/assets/20733831/3119b0f9-de3c-48e0-a33f-f026c4720dc0)